contracts/integrations/examples/contractCreator/withTreasury/ResolverTaskCreator.sol

```diff
@@ -0,0 +1,72 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+
+import "../../../OpsTaskCreator.sol";
+
+/**
+ * @dev
+ * Example contract that creates a resolver task.
+ */
+// solhint-disable not-rely-on-time
+// solhint-disable no-empty-blocks
+contract ResolverTaskCreator is OpsTaskCreator {
+    uint256 public count;
+    uint256 public lastExecuted;
+    bytes32 public taskId;
+    uint256 public constant MAX_COUNT = 5;
+    uint256 public constant INTERVAL = 3 minutes;
+
+    event CounterTaskCreated(bytes32 taskId);
+
+    constructor(address _ops, address _owner) OpsTaskCreator(_ops, _owner) {}
+
+    function createTask() external {
+        require(taskId == bytes32(""), "Already started task");
+
+        bytes memory resolverData = abi.encodeCall(this.checker, ());
+        bytes memory execSelector = abi.encode(this.increaseCount.selector);
+
+        ModuleData memory moduleData = ModuleData({
+            modules: new Module[](2),
+            args: new bytes[](2)
+        });
+
+        moduleData.modules[0] = Module.RESOLVER;
+        moduleData.modules[1] = Module.PROXY;
+
+        moduleData.args[0] = _resolverModuleArg(address(this), resolverData);
+        moduleData.args[1] = _proxyModuleArg();
+
+        bytes32 id = ops.createTask(
+            address(this),
+            execSelector,
+            moduleData,
+            address(0)
+        );
```

Comment:
maybe reorganize it like that to have better readability in module implementation & task creation

```suggestion
        ModuleData memory moduleData = ModuleData({
            modules: new Module[](2),
            args: new bytes[](2)
        });

        moduleData.modules[0] = Module.RESOLVER;
        moduleData.args[0] = _resolverModuleArg(address(this), abi.encodeCall(this.checker, ()));
        
        moduleData.modules[1] = Module.PROXY;
        moduleData.args[1] = _proxyModuleArg();
        
        bytes32 id = ops.createTask(
            address(this),
            abi.encode(this.increaseCount.selector),
            moduleData,
            address(0)
        );
```

Reply:
[1a637b0](https://github.com/gelatodigital/ops/pull/88/commits/1a637b04ec8568e7dd0d0044839aba42a69022e5)